sdks/python/hatchet_sdk/worker/worker.py

```diff
@@ -199,6 +233,18 @@ def _setup_loop(self) -> None:
         asyncio.set_event_loop(self.loop)
 
     async def _health_check_handler(self, request: Request) -> Response:
+        if self.config.enable_no_op_heartbeat_workflow:
+            ref = await self.admin_client.aio_run_workflow(
+                workflow_name="__internal__no_op_heartbeat_workflow",
+                input={},
+            )
```

Comment:
We really need to set the desired worker id here, but I'm not sure what a reasonable way to do that is since the worker doesn't know its own id?